src/widgets/span/lv_span.h

```diff
@@ -214,6 +221,37 @@ uint32_t lv_spangroup_get_expand_width(lv_obj_t * obj, uint32_t max_width);
  */
 int32_t lv_spangroup_get_expand_height(lv_obj_t * obj, int32_t width);
 
+/**
+ * Get the span's coords in the spangroup.
+ * @note Before calling this function, please make sure that the layout of span group has been updated.
+ * +--------+
+ * |Heading +--->------------------+
+ * |  Pos   |   |     Heading      |
+ * +--------+---+------------------+
+ * |                               |
+ * |                               |
+ * |                               |
+ * |            Middle   +--------+|
+ * |                     |Trailing||
+ * |                   +-|  Pos   ||
+ * |                   | +--------+|
+ * +-------------------v-----------+
+ * |     Trailing      |
+ * +-------------------+
```

Comment:
@vwheeler63 and @kdschlosser are working on making the API more Doxygen compliant, and I'm afraid that this explanation won't be processed correctly. Could you update it a simpler textual version?

Reply:
put the brakes on making any changes like this.

I will go over it with you in the video chat. You may or may not like it. we will see.

Reply:
@kisvegabor @kdschlosser 

Doxygen will preserve if it is indented 4 spaces from the line of '\*'.

Reply:
> Doxygen will preserve if it is indented 4 spaces from the line of '*'.

Good to know. It should be documented at some point for sure.


@W-Mai do we need to expose this function? If it can be moved to the C file, it wouldn't be processed by Doxygen and we had one less API function too. 🙂

Reply:
> If it can be moved to the C file, it wouldn't be processed by Doxygen and we had one less API function too.

Doxygen has great flexibility about what files are included or excluded....  In case it presents more options.

Reply:
> > Doxygen will preserve if it is indented 4 spaces from the line of '*'.
> 
> Good to know. It should be documented at some point for sure.
> 
> @W-Mai do we need to expose this function? If it can be moved to the C file, it wouldn't be processed by Doxygen and we had one less API function too. 🙂

I think this is an optional for user's to decide how to realize their event handler for the span `click/...`. so I think we should keep this api.